Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:edge
```

Comment:
Maybe it's better to go for [latest](https://github.com/alpinelinux/docker-alpine/blob/adebcfd075c83ef788b5f071678dcfb8ea118bb3/x86_64/Dockerfile), edge may contain bugs.

Reply:
Changed to alpine:latest